src/components/Form/FormProvider.tsx

```diff
@@ -64,7 +62,7 @@ type FormProviderProps<TFormID extends OnyxFormKey = OnyxFormKey> = FormProvider
     };
 
 type FormRef<TFormID extends OnyxFormKey = OnyxFormKey> = {
-    resetForm: (optionalValue: OnyxFormValues<TFormID>) => void;
+    resetForm: (optionalValue: FormOnyxValues<TFormID>) => void;
 };
 
 function FormProvider(
```

Comment:
Does `FormProvider` validate that its children contains `InputWrappers` for every `inputID` defined for the form?

Reply:
Unfortunately no, `FormProvider` doesn't validate that its children contain `InputWrappers` for every `inputID` defined for the form. 

This kind of validation has to be implemented on the runtime level as TS can't enforce certain children for a specific component at compile time.